lib/dispatch-rider/scheduled_job.rb

```diff
@@ -0,0 +1,59 @@
+require "active_record"
+
+# @note: Later this could be pulled out to its own gem and included depending on what ORM the user
+# prefers. Something like:
+# @example
+#  gem "dispatch_rider-active_record"
+#  gem "dispatch_rider-rom" # Ruby Object Mapper
+#  gem "dispatch_rider-mongo_mapper"
+#  gem "dispatch_rider-ohm"
+module DispatchRider
+  class ScheduledJob < ActiveRecord::Base
+    class << self
+      def publisher
+        @publisher ||= Publisher.new
+      end
+
+      # @param [ActiveSupport::Duration] every
+      # @example: DispatchRider::ScheduledJob.publish_due_jobs every 1.minute
+      def publish_due_jobs(every: nil)
+        loop {
+          claim_stub = get_new_claim_stub
+          due.unclaimed.update_all claim_stub
```

Comment:
because the unclaimed filter just modifies the sql .. this will only claim unclaimed ones .. yea this is ok ..